sample-install/snirouting_tls/overlays/minikube/server-certificate.yaml

```diff
@@ -0,0 +1,29 @@
+#
+# Copyright Kroxylicious Authors.
+#
+# Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+#
+
+---
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: server-certificate
+  namespace: kafka
+spec:
+  isCA: true
+  commonName: kroxy-server-cert
+  secretName: kroxy-server-tls
+  privateKey:
+    algorithm: RSA
+    size: 4096
+    encoding: PKCS8
```

Comment:
Note the cert-manager default is PKCS1 - that won't work with Kroxylicious unless you put Bouncy Castle in the classpath.  PKCS8 works fine.  cert-manager is also capable of writing a pk12 keystore too.

Reply:
Added as a comment into the certificate CR.